src/ssz/tree_view.zig

```diff
@@ -104,6 +130,22 @@ pub fn TreeView(comptime ST: type) type {
             out.* = self.data.root.getRoot(self.pool).*;
         }
 
+        /// Get the length of the array. For Vector returns compile-time constant, for List reads from tree.
+        pub fn getLength(self: *Self) !usize {
```

Comment:
make this compatible to ~~`setListLength()`~~ `updateListLength()` api
```suggestion
        pub fn getListLength(self: *Self) !usize {
```